src/libraries/Pool.sol

```diff
@@ -444,6 +441,7 @@ library Pool {
             self.feeGrowthGlobal1X128 = state.feeGrowthGlobalX128;
         }
 
+        // in an unchecked block so sign flipping is not safe without a helper function
```

Comment:
again, this feels over-commented to me, using the helper function is self explanatory